stories/wizard/wizard.stories.js

```diff
@@ -0,0 +1,868 @@
+import '../../dist/button.css';
+import '../../dist/icon.css';
+import '../../dist/wizard.css';
+
+export default {
+    title: 'Components/Wizard',
+    parameters: {
+        description: `The Wizard component divides into sections a long or unfamiliar task and guides the user through it. 
+The Wizard consists of the walkthrough screen, where the form sections are revealed subsequently after completing, and the summary page, where the form is displayed in read only mode for assessment and final submission.
+
+
+## Usage
+
+**Use the wizard if:**
+
+- the user has to accomplish a long or unfamiliar task
+- the flow consist of a minimum of 3 and a maximum of 8 steps
+
+**Do not use the wizard if:**
+
+- a task has only 2 steps
+- a task has more than 8 steps
+- the format of the task is familiar to the user
+
+## Layout
+The wizard can be used both in a full-screen layout as well as in the flexible column layout. There is no navigation from the wizard to a next page. After completion or cancellation, the user is always navigated to the page the wizard was triggered from.
+
+## Responsive paddings
+
+These modifier classes will add horizontal paddings to the content and can be applied on the <code>fd-wizard\\_\\_progress-bar</code> level and/or on the <code>fd-wizard\\_\\_content</code> level.
+
+
+|  &nbsp;&nbsp;**rem** &nbsp;&nbsp; |  &nbsp;&nbsp;**min-width** &nbsp;&nbsp; |  &nbsp;&nbsp;**max width** &nbsp;&nbsp; |  &nbsp;&nbsp;**modifier class** &nbsp;&nbsp; |
+| :----: | :----------: | :----------: | :----------------------------------: |
+| 1rem | _n/a_ | 599px | <code>fd-wizard\\_\\_progress-bar--s</code> / <code>fd-wizard\\_\\_content--s</code> |
+| 2rem | 600px | 1023px | <code>fd-wizard\\_\\_progress-bar--m</code> / <code>fd-wizard\\_\\_content--m</code> |
+| 2rem | 1024px | 1439px | <code>fd-wizard\\_\\_progress-bar--l</code> / <code>fd-wizard\\_\\_content--l</code> |
+| 3rem | 1440px | _n/a_ | <code>fd-wizard\\_\\_progress-bar--xl</code> / <code>fd-wizard\\_\\_content--xl</code> |
+
+
+## Step types
```

Comment:
I think we need to refer to it as ##Modifiers here

and add a ##Types section under the Usage section, with this:

"There are two types of wizard that offer different functionality: _**standard**_ and _**branching**_.

**Use the standard type if:**

- The total number of steps is known in advance.
- The number of steps does not change during usage.
- There is linear progression from one step to the next.

**Use the branching type if:**

- The total number of steps is not known.
- The number of steps may change during usage.
- There is non-linear progression. In other words, the user’s choice during one step determines which step comes next.
- In both types of wizard you can let users skip steps. Label these steps as “Optional”.